provider/resource_keycloak_openid_client.go

```diff
@@ -64,6 +64,11 @@ func resourceKeycloakOpenidClient() *schema.Resource {
 				Computed:  true,
 				Sensitive: true,
 			},
+			"client_authenticator_type": {
+				Type:      schema.TypeString,
+				Optional:  true,
+				Computed:  true,
```

Comment:
We should set the default value here instead of checking for an empty string and setting it within the `keycloak` package.

Reply:
Hey, sorry, didn't get notified of this reply. 
Fixed it now!